plugins/modules/cohesity_finalize_migration.py

```diff
@@ -235,10 +237,20 @@ def main():
         else:
             task_details["task_id"] = task_id
             response = finalize_migration(module, task_details)
-
+            count = 0
+            while count < 5:
+                status = get__restore_task_status__by_id(module, task_details)
+                if status == "Succeeded":
+                    module.exit_json(
+                        changed=True,
+                        msg="Succesfully completed the Cohesity Migration Job '%s'"
+                        % task_name,
+                    )
+                time.sleep(60)
```

Comment:
Why are these values hard coded, for each sync, start, cancel , finalize, we return just the task id and let the user poll using the other module, correct ?